storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -1459,5 +1425,143 @@ Item {
                         root.swapAdaptor.currencyStore.currentCurrency))
             closeAndVerfyModal()
         }
+
+        function test_modal_switching_networks_payPanel() {
+            let test_data = ["ETH", "aave"]
```

Comment:
You could put the `test_data` outside of the function into `function test_modal_switching_networks_payPanel_data()` and use that instead of the `for` loop

Reply:
See https://doc.qt.io/qt-5/qml-qttest-testcase.html#data-driven-tests